modules/core/src/test/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotStateChannelEventsProcessorSuite.scala

```diff
@@ -68,75 +73,13 @@ object GlobalSnapshotStateChannelEventsProcessorSuite extends MutableIOSuite {
       address = keyPair.getPublic().toAddress
       output <- mkStateChannelOutput(keyPair)
       snapshotInfo = mkGlobalSnapshotInfo()
-      service = mkProcessor(Set(address))
+      service = mkProcessor(Map(address -> output.snapshotBinary.proofs.map(_.id.toPeerId)))
       expected = (
         SortedMap((address, NonEmptyList.one(output.snapshotBinary))),
         SortedMap.empty[Address, (Option[Signed[CurrencyIncrementalSnapshot]], CurrencySnapshotInfo)],
         Set.empty
       )
-      result <- service.process(snapshotInfo, output :: Nil)
-    } yield expect.same(expected, result)
-
-  }
-
-  test("return two dependent sc events") { res =>
```

Comment:
what about these tests?

Reply:
These tests are now done in `GlobalSnapshotStateChannelAcceptanceManagerSuite`.